### PR TITLE
Cache gdb in CI

### PR DESCRIPTION
**Pull request by seelabs**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -59,6 +59,7 @@ cache:
   - $BOOST_ROOT
   - llvm-$LLVM_VERSION
   - cmake
+  - $GDB_ROOT
 
 before_install:
   - bin/ci/ubuntu/install-dependencies.sh
```
Comment (seelabs) [inline .travis.yml]: Yes, it does. (I thought I did that, but I only did it for circle ci, not travis). Thanks @mellery451. Will fix

**Comment by seelabs** [comment]
Thanks @ximinez. I pushed a commit to create the gdb directory even if it's not needed; I can't test this locally, so I'll keep an eye on CI to see if this fixes it.

**Comment by seelabs** [comment]
@ximinez I squashed down and confirmed the debug build is using the cache. The non-unity build failed, but that was a timeout (and I confirmed it correctly did _not_ try to build gdb for that target).

**Comment by ximinez** [comment]
Good build! I'm going to force rebuild the Travis debug to test the cache and then I'll pass it.

**Comment by seelabs** [comment]
@ximinez Was there something else that needs to be done for this? I think it's ready to be passed, but wanted to check with you first.

**Comment by seelabs** [comment]
In 0.80.0-b2

**Pull request closed without merging**